external/grpc/Kconfig.protocol

```diff
@@ -6,7 +6,6 @@
 config GRPC
 	bool "gRPC"
 	default n
-	depends on NET
-	depends on HAVE_CXX
+	depends on NET && HAVE_CXX && PROTOBUF
```

Comment:
1. Don't you need llvm? Is HAVE_CXX enough?
2. PROTOBUF is a simple config and doesn't have any dependancy. Let's use ```select PROTOBUF```.

Reply:
I'm confused that protobuf is a simple config. Is there any differences between HAVE_CXX and PROTOBUF. if there are differences, I think that we should guide the developers it first.